### ISSUE TITLE
Low response rate to the contributors reports

### ISSUE THREAD

**Issue by antoviaque** [openedx/wg-coordination#110]
Contributor meetup topic imported from https://openedx.atlassian.net/wiki/spaces/COMM/pages/3934289923/2023-11-28+CC+Working+Group+Meeting+Notes

**Low response rate to the contributors reports**

* Notes from previous meetings: 
    * See the [discussion in the last report](https://discuss.openedx.org/t/contributors-meetup-async-update-october-28th-2023-november-10th-2023/11644/3?u=antoviaque), only 25% of core contributor respond. Is it lack of interest/use? Lack of core contributor work in the first place?
    * Xavier: Complex topic, linked to a lot of other issues - in a nutshell:
       * @Ali Hugo performing core contributors interviews and preparing a survey about this
       * Some lack of interest in the outcome, not everything is interesting to all, need to refine the content, presentation & frequency
       * Synchronous vs async - reports use async tools, but most of the activity and culture of the project is on synchronous tools 
       * Lack of activity from a part of the core contributors - might need to establish a “alumni core contributor” role, to still recognize past work, while making it clear who is currently active
* Action items: @ali-hugo to post on the forum about [outcomes from first interviews](https://discuss.openedx.org/t/findings-from-interviews-with-core-contributors/11853) & discussion items for the survey

**Comment by ali-hugo** [comment]
Hi @cassiezamparini @antoviaque 

I'm interested to know the purpose of these "Contributors Coordination" issues; is the aim to improve visibility? If so —I'm all for it— but I wonder if there's a way to make the wording more clear. I'm often pinged on things and am unsure what I'm expected to do (unless I read very carefully).

Also, there seems to be an overlap between the content of these issues, and what could potentially go into the post-meeting notes instead. Do we need both?

**Comment by cassiezamparini** [comment]
@ali-hugo My understanding is visibility, and the fact that the CC Coordination group is the driving force behind these initiatives. But I see what you're saying. What may be better is that the "assignee" of a ticket on this board chat to the persons involved in whatever the "initiative" or "task" is, and then updates the task accordingly with minimal pings.

I agree though - getting random pings is noisy - and getting context can take time.

**Comment by cassiezamparini** [comment]
Thanks for clearing that up @antoviaque

**Comment by ali-hugo** [comment]
@antoviaque 
> It used to be a table on the list of agenda items on the wiki, but it was pretty cumbersome to maintain from one meeting to another

This is definitely an improvement to the table! 👏 

> What are the notifications that you would not want to receive precisely? Note that you can unsubscribe from notifications on individual tasks, or at the level of a repo.

I think it's less an issue of receiving too many notifications, and more an issue of receiving notifications for items that are not immediately clear. Take this issue for example: here are the things that were not clear to me when I first opened it:

- The title doesn't make it clear that the purpose of this issue is to follow up on a meeting topic, rather than to describe a specific task to be done
- The "action item" in the description is for me, but the the assignee of this issue is @cassiezamparini. That is a bit confusing
- Perhaps [this "update"](https://github.com/openedx/wg-coordination/issues/110#issuecomment-1956038188) should be given a more obvious title - something like "Update from Feb 20 Meeting"

In short, it's just not super obvious what this ticket is for. I think a few small tweaks would go a long way to making ithese tickets more clear.

Let me know if that all makes sense.

**Comment by antoviaque** [comment]
@ali-hugo OK, I'll see what I can do to improve these aspects when I do a pass over the board next.

**Comment by antoviaque** [comment]
@ali-hugo I just did a quick pass - I have added the mention "This issue is to follow up on a topic from the contributor meetup working group" at the top of the description of all the open tickets currently in the board (at least the ones I opened, I couldn't edit the other ones). For the current ticket, I have updated the description to remove your original action item - it was a copy-paste from the wiki page, and in the meantime the task passed on to @cassiezamparini so the assignee of the current task was more up to date. 

Does that help? Let me know if you notice other things like that.

**Comment by cassiezamparini** [comment]
@antoviaque This is still on hold. Waiting for results from the survey. You'll see from the latest [CC Check-in](https://discuss.openedx.org/t/contributors-meetup-async-update-march-2nd-march-15th-2024/12525/2) that we've had a 20% response rate on the survey so far.

**Comment by ali-hugo** [comment]
@antoviaque Adding that intro sentence makes the purpose of the issue much clearer to me. Thanks!
